_docs/tasks/istio-auth.md

```diff
@@ -58,30 +60,30 @@ boundary. This will offer strong security for Istio CA.
 * Services in the same Kubernetes cluster but different namespaces are able to talk to each other through Istio Auth
 without extra trust setup.
 
-#### Deplying CA
+#### Deploying CA
 
 The following command creates  namespace *istio-system* and deploys CA into the namespace:
 
 ```bash
-kubectl apply -f ./kubernetes/istio-auth/istio-cluster-ca.yaml
+kubectl apply -f templates/istio-auth/istio-cluster-ca.yaml
 ```
 
-#### <a name="istioconfig"></a>Enabling Istio Auth in Istio config
+#### Enabling Istio Auth in Istio config
 
-The following command uncomments the line *authPolicy: MUTUAL_TLS* in the file *kubernetes/istio-X.yaml*,
-and backs up the original file as *istio-X.yaml.bak*
-(*X* corresponds to the Kubernetes server version, choose "15" or "16").
+The following command uncomments the line *authPolicy: MUTUAL_TLS* in the file *istio.yaml*,
+and backs up the original file as *istio.yaml.bak*
 
 ```bash
-sed "s/# authPolicy: MUTUAL_TLS/authPolicy: MUTUAL_TLS/" ./kubernetes/istio-X.yaml > ./kubernetes/istio-auth-X.yaml
+mv templates/istio.yaml templates/istio.yaml.bak
+sed "s/# authPolicy: MUTUAL_TLS/authPolicy: MUTUAL_TLS/" templates/istio.yaml > templates/istio-auth.yaml
```

Comment:
Is this correct? There will be no `istio.yaml` after the first `mv` and the second commend will fail.